crates/test-programs/wasi-http-tests/src/bin/outbound_request.rs

```diff
@@ -57,6 +57,12 @@ fn request(
 
     let future_response = default_outgoing_http::handle(request, None);
 
+    let incoming_response = types::future_incoming_response_get(future_response)
+        .ok_or_else(|| anyhow!("incoming response is available immediately"))?
+        // TODO: maybe anything that appears in the Result<_, E> position should impl
+        // Error? anyway, just use its Debug here:
+        .map_err(|e| anyhow!("{e:?}"))?;
```

Comment:
I also ran into this bug in wasmtime's wit-bindgen the other day and never finished the PR with the fix, and I guess it is a bug in the guest wit-bindgen as well for the same reasons. We don't need to hold up this PR waiting on the fix, especially because this comment is just code motion - just caught my eye.